src/dataMap/metaManager/metaSchema.js

```diff
@@ -404,7 +404,7 @@ export default () => {
      * @example
      * ```js
      * cells: function(row, column, prop) {
-     *   const cellProperties = {};
+     *   let cellProperties = { readOnly: false };
```

Comment:
```suggestion
     *   const cellProperties = { readOnly: false };
```
The `cellProperties` props are changed by reference so `const` keyword is valid here. If the variable were reassigned below I would agree with this suggestion :)

Reply:
👌 dunno why I wanted to have it defined as `let`. Suggestion committed!